### PR TITLE
Consistency fixes

### PR DESCRIPTION
**Pull request by Carbonhell**
changes: 
- rscadd: "Adds a new autolathe called atmoslathe, found in Atmospherics, to make tanks and atmos-related items."
- rscadd: "Added a recipe for fire extinguisher cabinets, they can now be built with metal and attached to walls."
- rscadd: "Now you can click cablecuffs in hand to retrieve the wires back."
- rscadd: "Added the material value to some items,this means stuff like pneumatic cannons can now be recycled in autolathe."
- rscadd: "Adds a new autolathe called atmoslathe, found in Atmospherics, to make tanks and atmos-related items."
- rscadd: "Fire extinguishers now accept and can spray any chemical!"
- rscadd: "Huge revamp of tanks(Water and fuel tanks).Now they accept any chemical, they'll display it on the left and you can toggle the lid out to insert them in through a screwdriver. They'll also spray its contents everywhere if shot, and trigger an heat-reaction if welded."
- tweak: "Water/lube will make a floor slippable depending on how many units you use.One unit means it'll wear out easily, more than 10 means the old time."
  Also cleaned the code a bit by putting newscaster frame and button frame in the right file, and other stuff

### CODE REVIEW

**Comment by Carbonhell** [comment]
This is just the first part, i still have to start the other part

**Comment by danhibiki** [comment]
"rscadd: "Fire extinguishers now accept and can spray any chemical!""
No from me

File: code/game/objects/items/devices/object-in-a-box.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/obj/item/device/object_in_a_box
```
Comment (danhibiki) [inline code/game/objects/items/devices/object-in-a-box.dm:1]: new item? not listed in changelog, what else is in here?

Comment (Chronitonity) [inline code/game/objects/items/devices/object-in-a-box.dm:1]: It's defined right above this
+if(ispath(being_built.build_path, /obj/structure)) // if we're making a structure, make an object-in-a-box item that dispenses said structure on use, or everything runtimes

Comment (danhibiki) [inline code/game/objects/items/devices/object-in-a-box.dm:1]: Ah, couldn't see that far on just reviewing changes.

File: html/changelogs/Carbonhell-Consistency1.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+author: Carbonhell
+
+delete-after: True
+
+changes: 
+  - rscadd: "Adds a new autolathe called atmoslathe, found in Atmospherics, to make tanks and atmos-related items."
+  - rscadd: "Added a recipe for fire extinguisher cabinets, they can now be built with metal and attached to walls."
+  - rscadd: "Now you can click cablecuffs in hand to retrieve the wires back."
+  - rscadd: "Added the material value to some items,this means stuff like pneumatic cannons can now be recycled in autolathe."
+  - rscadd: "Adds a new autolathe called atmoslathe, found in Atmospherics, to make tanks and atmos-related items."
```
Comment (danhibiki) [inline html/changelogs/Carbonhell-Consistency1.yml]: Same line twice in changelog

File: code/modules/research/designs/autolathe_designs.dm [hunk 3]
```diff
@@ -617,4 +609,132 @@
...
+	build_path = /obj/item/weapon/pneumatic_cannon
+	category = list("hacked", "Atmos")
+
+/datum/design/rpd
+	name = "Rapid pipe dispenser (RPD)"
+	id = "rpd"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 75000, MAT_GLASS = 37500)
+	build_path = /obj/item/weapon/pipe_dispenser
+	category = list("hacked", "Atmos")
```
Comment (danhibiki) [inline code/modules/research/designs/autolathe_designs.dm:740]: Why hacked?

Comment (Chronitonity) [inline code/modules/research/designs/autolathe_designs.dm:740]: thats where it was before

Comment (danhibiki) [inline code/modules/research/designs/autolathe_designs.dm:740]: Ah I see, its a new entry because it was moved from construction to atmos.

Comment (Carbonhell) [inline code/modules/research/designs/autolathe_designs.dm:740]: Yea, it was a product of hacked autolathe,i though it was more fitting in the atmoslathe,i just moved it.

File: code/game/objects/items/devices/object-in-a-box.dm (hunk 1 above)
Comment (Carbonhell) [inline code/game/objects/items/devices/object-in-a-box.dm:1]: It's just an item needed for atmoslathe to dispense /obj/structure due to it runtiming to hell due to missing vars. This item is just the product of atmoslathe that,when used, deploys the "master" object(currently only used for chemtank.)

File: code/modules/research/designs/autolathe_designs.dm (hunk 3 above)
Comment (Chronitonity) [inline code/modules/research/designs/autolathe_designs.dm:740]: You realize you made the better cannon easier to build than the ghetto cannon right

**Comment by SplergeBucket** [comment]
Sounds good to me, the pneumatic cannon is very easy to build though, and as I recall very, very dangerous.

**Comment by Lawligagger** [comment]
Sounds good.

**Pull request merged**